apps/website/src/pages/docs/variables.mdx

```diff
@@ -2,17 +2,18 @@
 title: Variables
 description: Use iTwinUI-variables to integrate design tokens into your application.
 layout: ./_layout.astro
-toc: false
 ---
 
 import ColorList from '~/components/variables/ColorList.astro';
 import FontsDemo from '~/components/variables/FontsDemo.astro';
 import SizesDemo from '~/components/variables/SizesDemo.astro';
+import SizesInputDemo from '~/components/variables/SizesInputDemo.astro';
 import OpacityDemo from '~/components/variables/OpacityDemo.astro';
 import BorderRadiusDemo from '~/components/variables/BorderRadiusDemo.astro';
 import ShadowsDemo from '~/components/variables/ShadowsDemo.astro';
+import DurationDemo from '~/components/variables/DurationDemo.astro';
 
-## Installation
+<p>{frontmatter.description}</p>
```

Comment:
Replace frontmatter paragraph and the following paragraph/code with changes from [`648fbde` (#1335)](https://github.com/iTwin/iTwinUI/pull/1335/commits/648fbdea0479bee993882ef4b71d6da36b439269).

In that PR, I had it condensed to this:

> iTwinUI offers an extensive set of handcrafted CSS variables. If you're using iTwinUI-react, these variables will be automatically included and imported into your application. If you want to use it standalone, you can manually `npm install @itwin/itwinui-variables`.

Reply:
Done.